### PR TITLE
 [12.0] Migrate project_task_stage_allow_timesheet

### PR DESCRIPTION
**Pull request by ddufresne**

### CODE REVIEW

**Review by lmignon — APPROVED** [review]
@ddufresne Thank you for the migration. LGTM

**Comment by lmignon** [comment]
@benwillig Can you check this as you are the initial author?

File: project_task_stage_allow_timesheet/views/project_task_type.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <field name="arch" type="xml">
+
+            <field name="fold" position="before">
+                <field name="allow_timesheet"/>
+            </field>
+
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="project_task_type_tree_view">
```
Comment (tbaden) [inline project_task_stage_allow_timesheet/views/project_task_type.xml:20]: just cosmetical change (optional):
id before model
and use of original id (task_type_tree)

File: project_task_stage_allow_timesheet/views/project_task_type.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="project_task_type_form_view">
```
Comment (tbaden) [inline project_task_stage_allow_timesheet/views/project_task_type.xml:7]: just cosmetical change (optional):
id before model
and use of original id (task_type_edit)

File: project_task_stage_allow_timesheet/views/account_analytic_line.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <field name="arch" type="xml">
+
+            <field name="task_id" poistion="attributes">
+                <attribute name="domain"/>
+            </field>
+
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="account_analytic_line_tree">
```
Comment (tbaden) [inline project_task_stage_allow_timesheet/views/account_analytic_line.xml:20]: just cosmetical change (optional):
id before model
and use of original id (hr_timesheet_line_tree)

File: project_task_stage_allow_timesheet/views/account_analytic_line.xml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="hr_timesheet_line_form">
```
Comment (tbaden) [inline project_task_stage_allow_timesheet/views/account_analytic_line.xml:7]: just cosmetical change (optional):
id before model
and use of original id (hr_timesheet_line_form)

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM
if you want to, some optional changes:

**Review by benwillig — APPROVED** [review]

**Pull request merged**